0003-configuration-management-extension/README.md

```diff
@@ -49,20 +49,34 @@ The server should be able to listen for configuration request at the following r
 
 The payload should be a JSON-encoded object with the following fields:
 - `id` (required) - id of the message. Should be either string or number. Used in delivery confirmation process.
-- `configVersion` - current endpoint configuration version. Should be of integer type. Used by server to determine is it needed to send response with configuration, server must not send configuration record if latest available configuration version equals to the one sent by endpoint. If this field was excluded from the message, then server will send configuration anyway.
-- `config` - configuration body of an arbitrary JSON type.
+- `configVersion` - current endpoint configuration version. Should be of integer type.
```

Comment:
Please, specify all fields as either optional or required. (Also specify conditions when they can be omitted.)

You have removed this part now, so I'm not sure if the field is optional or not.